tests/model_registry/conftest.py

```diff
@@ -226,7 +227,10 @@ def model_registry_instance(
             teardown=teardown_resources,
         ) as mr:
             mr.wait_for_condition(condition="Available", status="True")
-            mr.wait_for_condition(condition="OAuthProxyAvailable", status="True")
+            if is_model_registry_oauth or get_product_version(admin_client=admin_client) >= Version.parse(
```

Comment:
Main is now pointing to 2.23. We should not work around testops image tag issue. They need to prioritize and fix it. We should not sign off otherwise.